index.html

```diff
@@ -92,190 +99,113 @@ <h3>LET'S CONNECT</h3>
         BY MICROVERSE ARE DONE , AND I'VE DONE IT IN A GOOD WAY , I'VE USED SECTION TAG TO CODE CARD
         PLEASE FOR THE NEXT CODE VIEWER HELP ME TO MOVE FORWARD , I'VE ONLY TRHEE ATTEMPS, BE AT MY PLACE ..,
     -->
+
+    <!---->
     <section id="grid" class="card-id">
-      <div class="card-one">
-        <img src="Icon and img/Tonic Portfolio.png" alt="" />
-        <h3 class="tonic">Tonic</h3>
-        <ul>
-          <li>
-            <h4>Canopy</h4>
-          </li>
-          <li>
-            <img src="Icon and img/Counter.png" alt="" />
-          </li>
-          <li>
-            <p>Band End Dev</p>
-          </li>
-          <li>
-            <img src="Icon and img/Counter.png" alt="" />
-          </li>
-          <li>
-            <p>2015</p>
-          </li>
-        </ul>
-        <div class="textTonic">
+     
+      <div class="template"> 
+        <div class="templateImg">
+          <img src="DeskImg/Snapshoot Portfolio.png" alt="">
+        </div>
+        <div class="templtaeDetails">
+          <h1>Tonic</h1>
+          <ul class="tempUl">
+              <li>Canopy</li>
+              <li>Back-end dev</li>
+              <li>2015</li>
+          </ul>
           <p>
             A daily selection of privately personalized reads; no accounts or
             sign-ups required.
           </p>
+          <ul class="tempLang">
+            <li class="luga">Html</li>
+            <li class="luga">Javascript</li>
+            <li class="luga">Css</li>
+        </ul>
+        <h3 class="seePro">See project</h3>
         </div>
-        <div class="language">
-          <ul>
-            <li>
-              <p>HTML</p>
-            </li>
-            <li>
-              <p>CSS</p>
-            </li>
-            <li>
-              <p>Javascript</p>
-            </li>
+        
+      </div> 
+
+      <div class="template"> 
+        
+        <div class="templtaeDetails">
+          <h1>Multi-post history</h1>
+          <ul class="tempUl">
+              <li>Canopy</li>
+              <li>Back-end dev</li>
+              <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+         
+            <ul class="tempLang">
+              <li class="luga">Html</li>
+              <li class="luga">Javascript</li>
+              <li class="luga">Css</li>
           </ul>
+      
+        <h3 class="seePro">See project</h3>
```

Comment:
- [ ] [OPTIONAL] Please I highly recommend making changes to the `seePro` class which is a `button` tag. In addition, I suggest adding hover effects to the `button`. 😄 
This is a screenshot of the hover effects of the `button` tag.

![Screenshot (278)](https://user-images.githubusercontent.com/78104544/236281215-5d5f394f-aff0-488e-9e49-73c37833b5f2.png)